components/AgencyBarChart.vue

```diff
@@ -1,16 +1,58 @@
 <template>
   <data-view :title="title" :title-id="titleId" :date="date">
+    <ul
+      :class="$style.GraphLegend"
+      :style="{ display: canvas ? 'block' : 'none' }"
+    >
+      <li v-for="(item, i) in items" :key="i" @click="onClickLegend(i)">
+        <button>
+          <div
+            :style="{
+              backgroundColor: colors[i].fillColor,
+              borderColor: colors[i].strokeColor,
+              width: '20px',
+            }"
+          />
+          <span
+            :style="{
+              textDecoration: displayLegends[i] ? 'none' : 'line-through',
+            }"
+            >{{ item }}</span
+          >
```

Comment:
閉じタグが途中で改行されてしまっています。

```suggestion
            >{{ item }}</span>
```